os/hal/ports/STM32/LLD/OTGv1/stm32_otg.h

```diff
@@ -33,7 +33,7 @@
 /**
  * @brief   OTG_HS FIFO memory size in words.
  */
-#define STM32_OTG2_FIFO_MEM_SIZE        1024
+#define STM32_OTG2_FIFO_MEM_SIZE        4096
```

Comment:
This is a fairly significant increase, does it need to be protected with a #define to conserve memory?

Reply:
This is for OTG HS only, doesn't impact any existing hardware, this is needed for better throughput on MSD